docs/posts/overview-of-ballerina-0.982.0.md

```diff
@@ -0,0 +1,321 @@
+---
+title: Overview of Ballerina 0.982.0
+author: Maryam Ziyad Mohamed
+date: 28 September 2018
+status: Published
+abstract: Introduction to New Features and Changes introduced with Ballerina 0.982.0
+socialmediaimage: //todo
+---
+
+# Improvements and New Features
+The Ballerina 0.982.0 release included new features along with improvements and bug fixes.
+
+## Language & Runtime
+
+### Introduction of the `channel` Type
+
+A new constrained type `channel`, that can be defined only as a top level node, has been introduced for communication 
+between parallel processes in Ballerina programs. Channels can be used for message correlation by sending and receiving 
+messages via different resources to the same channel and can also be used for inter-worker communication and worker 
+synchronization.
```

Comment:
Shall we change as follows:
"The new `channel` constrained type is introduced for communication 
between parallel processes in Ballerina programs. Channel can be defined only as a top level node.
You can use channels for message correlation by sending and receiving 
messages via different resources to the same channel. You can also use channels for inter-worker communication and worker 
synchronization."